.github/workflows/sync-dev-to-stable.yml

```
@@ -0,0 +1,239 @@
+      with:
+        fetch-depth: 0
+        token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Configure Git
+      run: |
+        git config --global user.name "github-actions[bot]"
+        git config --global user.email "github-actions[bot]@users.noreply.github.com"
+
+    - name: Check for differences between dev and stable
+      id: check-diff
+      run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:19:26: Double quote to prevent globbing and word splitting

(shellcheck)